### ISSUE TITLE
Merging two different runs may reflect in a samples separation in a downstream ordination?

### ISSUE THREAD

**Issue by MatS792** [benjjneb/dada2#1830]
Hi all,
I am using DADA2 to analyze 13 biofilms samples sequenced for the 16S rRNA V4 region.  
The samples were sequenced in two different periods. In detail 12 samples are sequenced together and 1 sample was sequenced one month later. Primers, sequencing company, Illumina seq., etc are the same. 

I used two different approaches to analyze them: 
1° approach : *Running dada2 on the two groups separately (12 together in a dada2 run and 1 alone in another dada2 run) and merging the two seqtabs before removing the chimeras*
2° approach : *Running dada2 on all the samples together*

In both the two approaches, the settings for filter and trimming, error learning, dada2, and taxa assignation were the same.
In terms of ASVs and reads, I got:
| | 1° approach | 2° approach |
| --------- | --------- | --------- | 
|ASVs | 6853 | 7115  |
|Reads | 2791892 | 2766477  |

Once I move in @phyloseq, after the normalization of the reads, I run an nMDS and what I find is this:
#### From the 1° approach
![first_apporach_github](https://github.com/benjjneb/dada2/assets/63846963/762da7a1-3ed2-4ab1-ac69-ccd0b7cda79a)

#### From the 2° approach
![second_apporach_github](https://github.com/benjjneb/dada2/assets/63846963/991ac1b6-fa38-449e-81dc-32a767b70ad5)

In synthesis: 
When I follow the 1° approach, the sample that was sequenced later, separate completely from the others. When, I follow the 2° approach this doesn't happen and the sample mix well with other similar samples.
The general idea is that when you get samples from different sequencing run, you should follow separated dada2 run and merge everything before the chimera removing. 

How much this thing is written in the stone?
Have you ever seen something like that?
Is there any reason behind this?

Thank you for your help.

**Comment by benjjneb** [comment]
Our general recommendation is to do as y ou did in the 2nd approach, run different amplicon sequencing batches through dada2 separately, and then merge them before chimera removal. This allows the appropriate error model for each batch to be learned and applied. If two batches have different error models for some reason, processing them all together leads to errors in inference.

It's a bit surprising to see such a large effect given that \

> Primers, sequencing company, Illumina seq., etc are the same.

but its often hard to know exactly how the same some steps are when they are performed months apart.

**Comment by MatS792** [comment]
Do you suggest to go ahead with the first approach even if it is less reccomended ?

**Comment by benjjneb** [comment]
Given your NMDS results, and the recommended way to treat data with DADA2, I would suggest the 2nd approach.

**Comment by MatS792** [comment]
I am sorry but I am getting confused. In the first answer are you referring to the 2nd approach as it is the 1st approach I described (running 2 different batches and then merging them before chimera removal) ?

**Comment by aimirza** [comment]
@MatS792 I would love for you to try running each run separately through dada2. Rerun both again separately but provide the resulting ASVs from both as priors to both, then compare the NMDS.

**Comment by g-huston** [comment]
Hello, apologies for asking a repeat question but I'm running into a similar situation with my samples (4 sets of eDNA samples each sequenced separately) and was confused reading through the responses. 
I should run each set separately to learn error rates, etc but then merge all 4 before chimera removal and taxonomy assignment, correct?

**Comment by benjjneb** [comment]
> I should run each set separately to learn error rates, etc but then merge all 4 before chimera removal and taxonomy assignment, correct?

That sounds right to me.

**Comment by g-huston** [comment]
Thank you so much!

**Comment by MichalOskiera** [comment]
I know that situation and recommendations well. 
Many times, investigated both approaches, and their effect on alpha / beta diversity in many analyses with many sequencing batches from same field (9 years) and same MiSeq machine, with different dada2 setting, sample-by-sample, pseudo-pool, pool true, and other variations.

If dada2 process samples with lower sequences depth, it will always result in lower alpha diversity.
Higher sequences pool (depth) always result in higher diversity after dada2.

If samples were sequenced in 2 batches - to make reliable comparison of that samples - we need to have similar sequencing depth (similar number of sequences) in a dada2 processing pool.

If You got 2 sequencing batches with unequal number of sequences, only methodology to compare such samples is to use approach 2: Run dada2 on all the samples pooled together - to have same sequences depth / diversity in all samples / batches.

We **sequenced samples in 2 batches**, in 1 batch there was lower sequences depth - less samples (dada pool true). I compared alpha diversity of that samples processed in 2 separate dada2 runs, after **merge tables, batch 1 resulted in very significant lower diversity**. Than, we sequenced same samples in one sequencing run together (1 batch), and processed in one dada2 run - all differences disappeared, alpha diversity were on same level. Than, I **processed in one dada2** run all mentioned samples (sequenced in 2 batches - 2 pools, and sequenced in 1 batch - same 2 pools together) - this **resulted in no alpha diversity differences**. By processing samples from 2 sequencing batches in 1 dada2 run (all samples together, pool=true), I obtained same diversity levels of all samples from 2 sequencing batches, as they were sequenced together.

My conclusions are: 
If batches comes from same machine, chemistry, **true error rate is less important than sequences depth / diversity during dada2 processing**.
If, I want to compare samples from different batches, I have to process them in one dada2 run. 

I know that this is against Benjamin recommendations, but always in my case - **merging separate dada2 tables, introduce much bigger bias, than poorly fit error model**.
